sdk/elastic/Azure.ResourceManager.Elastic/src/Generated/ElasticMonitorResource.cs

```diff
@@ -1281,9 +1281,9 @@ public virtual Response<ElasticTrafficFilterListResult> GetAllTrafficFilter(Canc
         /// </list>
         /// </summary>
         /// <param name="cancellationToken"> The cancellation token to use. </param>
-        public virtual async Task<Response<ElasticTrafficFilterListResult>> GetAssociatedTrafficFiltersAsync(CancellationToken cancellationToken = default)
+        public virtual async Task<Response<ElasticTrafficFilterListResult>> GetListAssociatedTrafficFilterAsync(CancellationToken cancellationToken = default)
```

Comment:
because operation id change? https://github.com/Azure/azure-sdk-for-net/blob/91af14e56d8b9f12d66fc5217e2e13670a21d8e1/sdk/elastic/Azure.ResourceManager.Elastic/src/autorest.md?plain=1#L117